nest-demo/docker/tempo.yml

```
@@ -0,0 +1,60 @@
+
+ingester:
+  max_block_duration: 5m # cut the headblock when this much time passes. this is being set for demo purposes and should probably be left alone normally
+  trace_idle_period: 1s
+  lifecycler:
+    ring:
+      replication_factor: 1
+
+distributor:
+  receivers:
+    jaeger:                  # the receives all come from the OpenTelemetry collector.  more configuration information can
+        protocols:           # be found there: https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)